services/api/src/models/TestData.ts

```diff
@@ -77,8 +77,39 @@ const basicQuest = new Quest({
   userid: 'testuser',
 });
 
+const basicPrivateQuest = new Quest({
```

Comment:
Recommend changing this to how we did the horror quest below, where it extends the base quest. Less code to update when we add a new field to the quest object!